installscripts/jazz-terraform-unix-noinstances/ecs_jenkins_task_definition.json

```diff
@@ -19,9 +19,9 @@
           "value": "${jenkins_user}"
         }
       ],
-    "memoryReservation": 2048,
+    "memoryReservation": 3072,
     "networkMode": "awsvpc",
-    "cpu": 2048,
+    "cpu": 1024,
```

Comment:
Shouldn't this be reading off of `variables.tf` as opposed to they being hard coded?

Reply:
Done.